x-pack/plugins/ml/server/routes/system.ts

```diff
@@ -226,4 +226,48 @@ export function systemRoutes(
       }
     })
   );
+
+  /**
+   * @apiGroup SystemRoutes
+   *
+   * @api {post} /api/ml/index_exists ES Field caps wrapper checks if index exists
+   * @apiName MlIndexExists
```

Comment:
You should also add this new route to `apidoc.json` to ensure it gets included in our API docs.

Reply:
Good catch - added in 3043dda0d0415412bb2811d81155ce6d818f1be0